AK/Format.cpp

```diff
@@ -17,10 +17,6 @@
 #include <string.h>
 #include <time.h>
 
-#if defined(AK_OS_SERENITY)
```

Comment:
No need to break compilation on serenity. The portability is a feature! A future port might end up working without much effort.

The testing change is fine tho

Reply:
Ok

Reply:
Updated to exclude this change